spec/line/bot/client_get_spec.rb

```diff
@@ -453,4 +464,22 @@ def generate_client
       ]
     )
   end
+
+  it 'get bot info' do
+    uri_template = Addressable::Template.new Line::Bot::API::DEFAULT_ENDPOINT + '/bot/info'
+    stub_request(:get, uri_template).to_return(body: BOT_INFO_CONTENT, status: 200)
+
+    client = generate_client
+    response = client.get_bot_info
+
+    json = JSON.parse(response.body, symbolize_names: true)
+    expect(json).to eq(
+      userId: 'Ufr47556f2e40dba2456887320ba7c76d',
+      basicId: '@abcdefghijklmn',
+      displayName: 'Example',
+      pictureUrl: 'https://obs.line-apps.com/abcdefghijklmn',
```

Comment:
I just want you to change this image URL.
example. https://example.com/hogehoge

Reply:
Sure.